packages/openapi-to-graphql/src/preprocessor.ts

```diff
@@ -155,6 +355,17 @@ export function preprocessOas(
         // Links
         const links = Oas3Tools.getEndpointLinks(path, method, oas, data)
 
+        // Callbacks containing [key: string]:PathItemObject
+        const callbacks = Oas3Tools.getEndpointCallbacks(
```

Comment:
I'm wondering if `processOperationCallbacks` should return its own operation array (instead of appending it to `data.operations`) and `callbacks` here should be set to that array instead of being set to `getEndpointCallbacks`, which essentially just returns the raw callback objects, if I understand correctly?

Reply:
Right, here `callbacks` are still raw.
I was also hesitating, but it seems safe to process them as operation.
After all, callbacks are composed of `OperationObject`, so why not ?

Reply:
It's a good possibility, then i think we would not need the `isSubscription` flag since subscriptions would be composed from data.callbacks instead of data.operations ( in `translateOpenAPIToGraphQL` ), so there would be no need to filter them.
But i think keeping this treatment in a separate function like `processOperationCallbacks` allow to keep things clear in `preprocessOas`.